RobotsWantedLeague/Services/RobotsService.cs

```diff
@@ -1,59 +1,50 @@
+using Microsoft.Extensions.Options;
+using MongoDB.Driver;
+
 namespace RobotsWantedLeague.Services;
 
 using RobotsWantedLeague.Models;
 
 public class RobotsService: IRobotsService
 {
-    private readonly List<Robot> robots;
-    private int idGenerator = 0;
-    public List<Robot> Robots { get => robots; }
+    private readonly IMongoCollection<Robot> robotsCollection;
+
+    public List<Robot> Robots { get => robotsCollection.AsQueryable().ToList(); }
 
-    public RobotsService()
+    public RobotsService(IOptions<RobotsDatabaseSettings> bookStoreDatabaseSettings)
     {
-        robots = new List<Robot>();
-    }
+        var mongoClient = new MongoClient(
```

Comment:
This is how we manage to build the `IMongoCollection`.  The only logic here is googlin g + copy/pasting